questions-js1/4.md

```diff
@@ -2,5 +2,6 @@ Q: What is the difference between defining and calling a function??
 
 A: {YOUR ANSWER HERE}
 
+declare and define are the same, and they mean when you write all the code for your function. At that point the function just sits there doing nothing. call is when you tell the JavaScript interpreter to run the code in your function.
```

Comment:
Defining a function is different from “calling a function.” When you “call” a function, you just write the name of the function followed by (); like this: nameOfFunctionToRun(); This will actually run the code inside that function, whereas defining the function does not run the code. It just defines the name.